react/components/EXPERIMENTAL_Table/Toolbar/ButtonGroup.tsx

```diff
@@ -41,21 +42,26 @@ const getButton = (type: ButtonType, props: Props) => {
   switch (type) {
     case 'columns': {
```

Comment:
And the `ButtonType` enum could be used here.

Reply:
Sure